quarkchain/cluster/master.py

```diff
@@ -1270,6 +1270,7 @@ def handle_new_root_block_header(self, header, peer):
         self.root_state.write_committing_hash(r_block.header.get_hash())
 
         adjusted_diff = await self.__adjust_diff(r_block)
+
```

Comment:
will remove in the next commit (if any).

Reply:
remove it!